qgepqwat2ili/tests/test_qgep.py

```diff
@@ -240,64 +240,63 @@ def test_case_f_export_selection_sia405(self):
             6,
         )
 
-    # test for complete VSA-DSS 2015 export, labels_orientation not set, should be optional
```

Comment:
Test failed because of missing adaption of wrong naming of fk_chute - see datamodel change https://github.com/QGEP/datamodel/pull/215
How to test?
- temporary adding datamodel change in hotfix?
- ???